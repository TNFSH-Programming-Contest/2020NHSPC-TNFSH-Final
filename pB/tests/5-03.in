4
655.846 760.702
145.953 760.702
145.953 -40.721
655.846 -40.721
638.751 747.276
