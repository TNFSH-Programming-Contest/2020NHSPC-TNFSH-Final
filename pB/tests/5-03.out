2713673.619
